src/wp-includes/rest-api/endpoints/class-wp-rest-post-types-controller.php

```diff
@@ -385,6 +390,12 @@ public function get_item_schema() {
 						),
 					),
 				),
+				'icon'           => array(
+					'description' => __( 'The icon for the post type.' ),
+					'type'        => 'string',
```

Comment:
This value can be string or null?

Reply:
It seems so if the post type hasn't declared a `menu_icon`. Should we update the `type` here? Do we need conditional adding to the response if the `menu_icon` is set? How do you handle this in general in core?

Reply:
You can have multiple types. See 
https://github.com/WordPress/wordpress-develop/blob/61f7e758952dabbe7e00cd2d9eeca2e7d82e4604/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L2218